### PR TITLE
DBT-915 added workspace filter

### PR DESCRIPTION
**Pull request by kayiganwa04**
## Context

We are adding workspace auto-complete in student materials and recording

[Add Workspace Filter in Student Materials](https://www.notion.so/tianlu/Add-Workspace-Filter-in-Student-Materials-a848f73c1d9146f79b39dfffbc939de9?pvs=4)

## Test

- Login as a student and go to materials and from there you can now filter by workspace

## Proof of Concept
image

### CODE REVIEW

File: src/services/LessonRepository.js [hunk 1]
```diff
@@ -461,6 +461,7 @@ export default {
    * @param {string} type
    * @param {string} tags[0]
    * @param {string} lessons[0]
+   * @param {string} workspaces[0]
```
Comment (abelbordonado) [inline src/services/LessonRepository.js:464]: Why there is [0]on this documentation?

**Comment by abelbordonado** [comment]
Now workspaces appear in the Admin -> Lesson -> Material, but is not filtering. Either fix it either hide it (i think before was hidden)

image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
image

**Comment by abelbordonado** [comment]
Solve the conflicts :D

**Review by abelbordonado — CHANGES_REQUESTED** [review]
Requested changes

**Comment by kayiganwa04** [comment]
> Now workspaces appear in the Admin -> Lesson -> Material, but is not filtering. Either fix it either hide it (i think before was hidden)
> 
> image

@abelbordonado  this was not hidden before you can even test on main I saw it haven't touched this one in this PR

**Pull request merged**